core/models/QuizModels.py

```diff
@@ -0,0 +1,47 @@
+from django.db import models
+
+
+class Answer(models.Model):
+
+    answer_text = models.CharField(max_length=100)
+    answer_text.description = "The full answer text to be displayed to the user"
+
+    answer_phrase = models.CharField(max_length=10)
+    answer_phrase.description = "Short one word summary of the answer to be used by the system"
+
+    def __str__(self):
+        return "{phrase}: {text}".format(phrase=self.answer_phrase, text=self.answer_text)
```

Comment:
Literal String Interpolation is a bit cleaner (Python 3.6+)
`return f'{self.answer_phrase}: {self.answer_text}'`

Reply:
https://dbader.org/blog/python-string-formatting

Reply:
I keep forgetting this is a thing. Fixed